Status gateways

sbeaver-netgate opened the pull request:
Two tables eliminated
Colors defined to eliminate magic
Bootstrap column width specified
All obsolescent classes and elements removed

sbeaver-netgate committed:
status_gateways.php Bootstrap conversion

sbeaver-netgate committed:
Corrected background color

sbeaver-netgate committed:
status_gateway_groups c- conversion complete

Eliminated outer table
Changed inner table parameters
Eliminated echo & print statements
Adjusted indenting and formatting

sbeaver-netgate committed:
status_gateways.php conversion complete

Two tables eliminated
Colors defined to eliminate magic
Bootstrap column width specified
All obsolescent classes and elements removed

sbeaver-netgate committed:
Added thead/tbody tabs and removed unneeded div id

SjonHortensius commented on usr/local/www/status_gateway_groups.php:
```diff
@@ -54,113 +54,117 @@
+	<table class="table table-hover table-striped table-condensed">
 		<tr>
-		  <td>
+			<th class="col-md-1"><?=gettext("Group Name"); ?></th>
```
What's the reason you added column size-classes here? I'm pretty sure they're not needed

SjonHortensius commented on usr/local/www/status_gateway_groups.php:
```diff
@@ -54,113 +54,117 @@
+<?php
+							} else {
+?>
+						<td>&nbsp;
```
This `&nbsp;` isn't needed

sbeaver-netgate committed:
status_gateways updated as requested

nbsp was not found. Perhaps I forgot to sync
Column widths were removed. You were right :)

SjonHortensius commented:
You added status_gateway_groups.php to this PR as well; I think that's accidental, can you revert that?

sbeaver-netgate commented:
I will attempt to do that.

## Steve

Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com | www.netgate.com 

> On Apr 26, 2015, at 6:13 AM, SjonHortensius notifications@github.com wrote:
> 
> You added status_gateway_groups.php to this PR as well; I think that's accidental, can you revert that?
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/SjonHortensius/pfsense/pull/137#issuecomment-96360089.

sbeaver-netgate committed:
Revert "status_gateways updated as requested"

This reverts commit ce328d51da0fffff92a6782295713e90267fa0aa.

sbeaver-netgate committed:
status_gateways.php reverted, recommitted

Reverted and re-committed as requested.
